docs/reference/action_list.rst

```diff
@@ -801,3 +841,77 @@ Combine this with configuring a custom template and you'll have a list column fu
 
 .. _`SonataDoctrineORMAdminBundle Documentation`: https://docs.sonata-project.org/projects/SonataDoctrineORMAdminBundle/en/4.x/reference/list_field_definition/
 .. _`here`: https://github.com/sonata-project/form-extensions/tree/1.x/src/Type
+
+Advance Usage
+-------------
+
+Displaying sub entity properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you need to display only one field from a sub entity or embedded object in a dedicated column, you can simply use the dot-separated notation::
+
+    namespace App\Admin;
+
+    use Sonata\AdminBundle\Admin\AbstractAdmin;
+    use Sonata\AdminBundle\Form\FormMapper;
+    use Sonata\AdminBundle\Datagrid\DatagridMapper;
+    use Sonata\AdminBundle\Datagrid\ListMapper;
+    use Sonata\AdminBundle\Show\ShowMapper;
+
+    final class UserAdmin extends AbstractAdmin
+    {
+        protected function configureListFields(ListMapper $list): void
+        {
+            $list
+                ->addIdentifier('id')
+                ->addIdentifier('firstName')
+                ->addIdentifier('lastName')
+                ->addIdentifier('address.street')
+                ->addIdentifier('address.ZIPCode')
+                ->addIdentifier('address.town')
+            ;
+        }
+    }
+
+.. note::
+
+    This only makes sense when the prefix path is made of entities, not collections.
+
```

Comment:
I think I would add this at the beginning with the first sentence

Reply:
Hey @franmomu I putted this bellow of 

```
Displaying sub model properties
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

please look if it's ok!

Reply:
<img width="724" alt="Screenshot 2022-01-17 at 13 33 44" src="https://user-images.githubusercontent.com/6358755/149807487-d4352622-2789-44b1-af18-4582f2824c74.png">

Reply:
<img width="724" alt="Screenshot 2022-01-17 at 13 33 44" src="https://user-images.githubusercontent.com/6358755/149807645-198444e4-8e84-4594-ac42-9df4ee74d3de.png">